oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/base/MetricsEsDAO.java

```diff
@@ -52,50 +52,49 @@ protected MetricsEsDAO(ElasticSearchClient client,
 
     @Override
     public List<Metrics> multiGet(Model model, List<Metrics> metrics) {
-        Map<String, List<Metrics>> groupIndices
-            = metrics.stream()
-                     .collect(
-                         groupingBy(metric -> {
+        Map<String, List<Metrics>> groupAliasIndices = new HashMap<>();
+        Map<String, List<Metrics>> groupIndices = new HashMap<>();
```

Comment:
First of all, we don't need to create two Map(s) always. For a particular model, it either uses `alias` or `mget`.  `model#isTimeRelativeID()` is the condition, so, wouldn't be affected by the metrics list.

Also, I noticed, that for alias usage, there is no point to use a Map, as you are going to have one alias(`IndexController.INSTANCE.getTableName(model)` returns consistent).

Reply:
You could use `if/else` or `return` fast in these two cases to avoid unnecessary costs.